.github/workflows/add-files-changed-label.yml

```
@@ -0,0 +1,165 @@
+            DESCRIPTION="PR changes $FILES_CHANGED files"
+          fi
+          
+          echo "Determined label: $LABEL with color: $LABEL_COLOR"
+          
+          # Get all labels in the repository to check if our label exists
+          echo "Checking if label exists in repository..."
+          ALL_LABELS_RESPONSE=$(curl -s -X GET \
+            -H "Authorization: token $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/labels")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)